elfi/methods/parameter_inference.py

```diff
@@ -745,12 +745,12 @@ def __init__(self, model, target_name=None, batch_size=1, initial_evidence=None,
         target_model : GPyRegression, optional
         acquisition_method : Acquisition, optional
             Method of acquiring evidence points. Defaults to LCBSC.
-        acq_noise_cov : float, or np.array of shape (n_params, n_params), optional
+        acq_noise_cov : float, or np.array of shape (n_params,) or np.array of shape (n_params, n_params), optional
```

Comment:
Should have just the types after the colon to follow the NumPy doc specification, that way autocomplete features and suggestions in IDEs work properly. The dimensions can be explained on the line below.